### PR TITLE
Timeline style 2

### PR DESCRIPTION
**Pull request by MilanBarande**

### CODE REVIEW

File: assembl/static2/css/components/timeline.scss [hunk 1]
```diff
@@ -114,13 +114,26 @@
     }
   }
 
-  .txt-active {
+  .txt-active-light {
+    .timeline-link {
+      color: $first-color;
+      font-family: $font-family4;
+      &:hover {
+        font-family: $font-family
```
Comment (swergas) [inline assembl/static2/css/components/timeline.scss]: ;

File: assembl/static2/css/components/timeline.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .timeline {
   height: 56px;
```
Comment (paolina92) [inline assembl/static2/css/components/timeline.scss]: Use `$navbar-height`

File: assembl/static2/js/app/components/debate/navigation/timelineSegment.jsx [hunk 3]
```diff
@@ -58,7 +59,15 @@ class TimelineSegment extends React.Component {
     }
   }
   render() {
-    const { index, barWidth, identifier, isCurrentPhase, isStepCompleted, phaseIdentifier, title, locale } = this.props;
+    const { index, barWidth, isCurrentPhase, isStepCompleted, title, locale } = this.props;
+    const timelineClass = () => {
```
Comment (paolina92) [inline assembl/static2/js/app/components/debate/navigation/timelineSegment.jsx]: use `import classNames from 'classnames'`. See an example in `components/debate/common/ideasLevel`

File: assembl/static2/css/components/timeline.scss (hunk 1 above)
Comment (vincentfretin) [reply assembl/static2/css/components/timeline.scss]: We should add scss-lint in pre-commit ;) But there is lots of warnings currently in our scss files.

**Comment by MilanBarande** [comment]
@paolina92 j'ai fait les changements demandés, bon pour merge?

**Pull request merged**